python/shipDigiReco.py

```diff
@@ -931,19 +931,27 @@ def findTracks(self):
 #check
     atrack = entry[1]
     theTrack = entry[0]
-    try:  theTrack.checkConsistency()
-    except:  print('Problem with track before fit, not consistent',atrack,theTrack)
+    try:
+      theTrack.checkConsistency()
+    except:
+      print('Problem with track before fit, not consistent',atrack,theTrack)
+      error = ROOT.genfit.Exception
+      ut.reportError(error)
```

Comment:
```suggestion
      ut.reportError(e)
```

Reply:
Not sure whether `reportError` can handle an exception, maybe it's better to just pass it a string.
I found out about that method yesterday...